main.tf

```diff
@@ -1,13 +1,58 @@
-resource "random_integer" "example" {
+resource "kubernetes_service_account" "service_account" {
   count = module.this.enabled ? 1 : 0
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+  metadata {
+    name        = var.deployment_name
+    namespace   = var.kubernetes_namespace
+    annotations = var.service_account_annotations
   }
 }
 
-locals {
-  example = format("%v %v", var.example, join("", random_integer.example[*].result))
-}
+resource "kubernetes_deployment" "tfc_cloud_agent" {
+  count = module.this.enabled ? 1 : 0
+
+  metadata {
+    name      = var.deployment_name
+    namespace = var.kubernetes_namespace
+    labels    = var.labels
+  }
+  spec {
+    selector {
+      match_labels = var.selector_match_labels
+    }
+    replicas = var.replicas
+
+    template {
+      metadata {
+        labels      = var.labels
+        annotations = var.deployment_annotations
```

Comment:
Everywhere else, we don't prefix with `deployment_` wondering if we should keep it here?

Reply:
Aha.. I see now

![image](https://user-images.githubusercontent.com/52489/95503821-b9951e00-0960-11eb-9665-ba87ecf78209.png)